tpl/bootstrap-compact.php

```diff
@@ -5,7 +5,7 @@
 		<meta http-equiv="X-UA-Compatible" content="IE=edge">
 		<meta name="viewport" content="width=device-width, initial-scale=1">
 		<meta name="robots" content="noindex" />
-		<title><?php echo i18n::_('PrivateBin'); ?></title>
+		<title><?php echo PrivateBin\i18n::_('PrivateBin'); ?></title>
```

Comment:
It should work when added on top of each template but I also thought about simple function helper (e.g. `_t()`)